lib/rubocop/config_obsoletion.rb

```diff
@@ -31,6 +31,7 @@ class ConfigObsoletion
       'Lint/UnneededSplatExpansion' => 'Lint/RedundantSplatExpansion',
       'Naming/UncommunicativeBlockParamName' => 'Naming/BlockParameterName',
       'Naming/UncommunicativeMethodParamName' => 'Naming/MethodParameterName',
+      'Naming/VariableNumber' => 'Naming/IdentifierNumber',
```

Comment:
IMHO, 1.0 stable may require a (new) mechanism that can provide a migration period to users for deprecated cop name until 2.0 is reached.

Reply:
Yeah, I totally agree. It'd be nice to have a mechanism which allows us to change the name of a cop without breaking configuration. E.g. cops can have a primary name and a deprecated name or something like this.